packages/blocks/src/lib/navigation-map/components/map.svelte

```diff
@@ -36,6 +36,23 @@ const toggleTileset = () => {
 let didHoverTooltip = Boolean(
   localStorage.getItem('navigation-service-card-tooltip-hovered')
 );
+
+let isFollowingBase = false;
+const followBase = () => {
+  if (baseGeoPose && isFollowingBase) {
+    map?.setCenter(baseGeoPose);
+    requestAnimationFrame(followBase);
+  }
+};
+
+const startFollowingBase = () => {
+  isFollowingBase = true;
+  requestAnimationFrame(followBase);
+};
+
+const stopFollowingBase = () => {
+  isFollowingBase = false;
+};
```

Comment:
This could definitely be consolidated to `toggleFollowingBase` but if I can figure it out we want to `stopFollowingBase` when the user drags the map.

Reply:
Could also do a little reactive helper:

```ts
$: toggleFollowingBase = isFollowingBase ? stopFollowingBase : startFollowingBase
```